containers/ecr-viewer/src/app/data/db/postgres_db.ts

```diff
@@ -44,3 +44,20 @@ export const getDB = (): IDatabaseScope => {
     };
   });
 };
+
+/**
+ * Performs a health check on the PostgreSQL database connection.
+ * @returns The status of the postgres connection or undefined if missing environment values.
+ */
+export const postgresHealthCheck = async () => {
+  if (!process.env.DATABASE_URL) {
+    return undefined;
+  }
+  try {
+    await getDB().database.connect();
```

Comment:
do we need to close this?

[docs](https://vitaly-t.github.io/pg-promise/Database.html#connect) make me think we need to call `done`, but I'm not super familiar with pg-promise